scripts/alignment_db_scripts/create_alignment_db_sharded.py

```diff
@@ -83,7 +84,7 @@ def create_index_default_dict() -> dict:
 
 
 def create_shard(
-    shard_files: List[Path], output_dir: Path, output_name: str, shard_num: int
+    shard_files: list[Path], output_dir: Path, output_name: str, shard_num: int
 ) -> dict:
```

Comment:
Perhaps having a [type alias](https://docs.python.org/3/library/typing.html) for the index dict could be helpful. 

If the index file structure is also used in the main library code, then I would consider adding the alias to the main openfold library. But this can also be done in a later PR.